web/src/App.tsx

```diff
@@ -22,6 +18,10 @@ import trackEvent from 'utils/analytics';
 
 const MapWrapper = lazy(async () => import('features/map/MapWrapper'));
 const LeftPanel = lazy(async () => import('features/panels/LeftPanel'));
+const LegendContainer = lazy(async () => import('components/legend/LegendContainer'));
+const FAQModal = lazy(() => import('features/modals/FAQModal'));
+const InfoModal = lazy(() => import('features/modals/InfoModal'));
+const SettingsModal = lazy(() => import('features/modals/SettingsModal'));
```

Comment:
Only one of these is `async`. Intentional?

Reply:
Good catch, it is not!